main.tf

```diff
@@ -20,16 +20,16 @@ locals {
 }
 
 resource "azurerm_storage_account" "storage_account" {
-  name                      = local.storage_account_name
-  resource_group_name       = var.resource_group_name
-  location                  = var.location
-  account_kind              = var.account_kind
-  account_tier              = var.account_tier
-  account_replication_type  = var.account_replication_type
-  access_tier               = var.access_tier
-  enable_https_traffic_only = var.enable_https_traffic_only
-  min_tls_version           = "TLS1_2"
-  allow_blob_public_access  = var.allow_blob_public_access
+  name                            = local.storage_account_name
+  resource_group_name             = var.resource_group_name
+  location                        = var.location
+  account_kind                    = var.account_kind
+  account_tier                    = var.account_tier
+  account_replication_type        = var.account_replication_type
+  access_tier                     = var.access_tier
+  enable_https_traffic_only       = var.enable_https_traffic_only
+  min_tls_version                 = "TLS1_2"
+  allow_nested_items_to_be_public = var.allow_nested_items_to_be_public
```

Comment:
you haven't updated the variable in variables.tf

Reply:
thanks.
variable allow_blob_public_access updated to allow_nested_items_to_be_public